src/components/Modals/ShareModal/ShareModal.container.ts

```diff
@@ -0,0 +1,31 @@
+import { Dispatch } from 'redux'
+import { connect } from 'react-redux'
+import { RootState } from 'modules/common/types'
+// import { getData as getDeployments } from 'modules/deployment/selectors'
+import { MapStateProps, MapDispatchProps, OwnProps } from './ShareModal.types'
+import DeployModal from './ShareModal'
```

Comment:
Rename to ShareModal and move to bottom of the import list
As a convention we place the relative imports at the end